pyproject.toml

```diff
@@ -95,13 +95,90 @@ dev = [
     "types-psutil==5.8.23",
     "types_requests==2.31.0.20240406",
     "types-protobuf==5.26.0.20240422",
+    "deptry==0.14"
 ]
 build = [
     "pyinstaller==6.6.0",
     "setuptools==69.5.1",
     "build==1.2.1"
 ]
 
+[tool.deptry]
+extend_exclude = [
+    "doc",
+    "scripts",
```

Comment:
I think we should include these (and adjust the exlusions below).

Reply:
When running deptry on `scripts`, it identifies a DEP001 violation: 
```
scripts/profile-time.py:46:8: DEP001 'humanize' imported but missing from the dependency definitions
Found 1 dependency issue.
```
If we want to include `scripts/`, we should add `humanize` to `pyproject.toml` instead of adjusting the exclusions.

Reply:
yes, sorry, I mean to not exclude what's required in the scripts (good we already found one issue here :))